var/spack/repos/builtin/packages/py-mdanalysis/package.py

```diff
@@ -45,7 +46,7 @@ class PyMdanalysis(PythonPackage):
     depends_on("py-numpy@1.21.0:", when="@2.5.0:", type=("build", "run"))
     depends_on("py-numpy@1.20.0:", type=("build", "run"))
 
-    depends_on("py-biopython@1.80:", type=("build", "run"))
+    depends_on("py-biopython@1.80:", when="@:2.6.1", type=("build", "run"))
     depends_on("py-networkx@2.0:", type=("build", "run"))
```

Comment:
Networkx is also no longer required. 

I would create a new "historical dependencies" section and move these down there so the current deps match pyproject.toml, but this is optional.

Reply:
Good catch. We are trying to remove mandatory dependencies, so I will do as you suggest to clarify things.